src/main/java/io/openliberty/tools/intellij/actions/RefreshLibertyToolbar.java

```diff
@@ -60,8 +61,9 @@ public static void refreshDashboard(Project project) {
         Component existingTree = null;
         Component existingActionToolbar = null;
         for (Component comp : components) {
-            if (comp.getName() != null && comp.getName().equals(Constants.LIBERTY_TREE)) {
-                existingTree = comp;
+            if (comp.getName() != null && comp.getName().equals(Constants.LIBERTY_SCROLL_PANE)) {
+                JBScrollPane scrollPane = (JBScrollPane) comp;
```

Comment:
Before doing typecasting it is always better to check
if (comp instanceof JBScrollPane) / if (scrollPane.getViewport().getView() instanceof Tree)

else there is a chance of JavaTypecastException.